src/serlio/prtMaterial/MaterialNameSource.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "maya/MGlobal.h"
+#include "maya/MStatus.h"
```

Comment:
these two includes could be moved to the `.cpp` file

Reply:
(obsoleted)